docs/Installation.md

```diff
@@ -46,27 +46,53 @@ If your Python environment doesn't include `pip`, see these
 [instructions](https://packaging.python.org/guides/installing-using-linux-tools/#installing-pip-setuptools-wheel-with-linux-package-managers)
 on installing it.
 
-To install dependencies, go into the `python` subdirectory of the repository,
+To install dependencies, **go into the `python` subdirectory** of the repository,
 and run from the command line:
 
     pip3 install .
 
+## Setting up ML-Agent within Unity
```

Comment:
This should be its own page.

Where do we plan to place the Quick Start to training?

Reply:
The reason I think this should belong here is because I think this is in practical a "installation" step, since we won't be able to use it without doing the step "Setting up ML-Agent within Unity", and the step "Setting up TensorflowSharp Support" (maybe mark this optional). Then the step "Play an example environment using pretrained model" would be a validation step on making sure everything has installed correctly. 

I didn't include the training in here for two reasons:

1. It would be a duplicate page for Getting-Started-With-Balance-Ball.md page. 
2. The process for training is too long, and it would no longer be a quick start.